README.md

```diff
@@ -7,22 +7,26 @@ SPDX-License-Identifier: CC-BY-4.0
 # MewBot
 
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
+
```

Comment:
Adding these breaks in does mean that the widgets end up not in a row: [before](https://github.com/mewler/mewbot/tree/dcc0c549503a6d6c08815f71e2998a05149f14a9#mewbot) | [after](https://github.com/mewler/mewbot/tree/7c23024d1fdecf433bd335df735df7aaa69b796d#mewbot).

It is only now that I have noticed that ther all contributors comments are already doing that for that one badge. I'd prefer them in a line but this does make it look less weird than it currently does.

Reply:
Fair enough! I'd assumed this was a typo and the intent was a collumn - as there seemed to be a mix before.